README.md

```diff
@@ -498,6 +503,184 @@ Sample result:
 - `safe`: Ensures the resource hasn't been modified in the meanwhile if `last_modified` is provided (default: `false`);
 - `last_modified`: The last timestamp we know the resource has been updated on the server.
 
+### Creating a group
+
```

Comment:
It may be worth shortly describing what a *group* is here, maybe linking to the kinto docs.